src/server/server_family.cc

```diff
@@ -1976,6 +2057,41 @@ void ServerFamily::ReplicaOf(CmdArgList args, ConnectionContext* cntx) {
   }
 }
 
+void ServerFamily::ReplicaOf(CmdArgList args, ConnectionContext* cntx) {
+  ReplicaOfInternal(args, cntx, true, true);
+}
+
+void ServerFamily::Replicate(string& host, string& port, bool* success) {
+  io::StringSink sink;
+  ConnectionContext ctxt{&sink, nullptr};
+
+  vector<MutableSlice> vargs{
+      {host.data(), host.size()},
+      {port.data(), port.size()},
+  };
+
+  CmdArgList args{vargs.data(), vargs.size()};
+
+  // we don't flush the database as the context is null
+  // (and also because there is nothing to flush)
+  ReplicaOfInternal(args, &ctxt, false, false);
```

Comment:
This kind of call site is confusing for readers. I'd recommend defining enums for better readability, like `enum class FlushDb { kFlush, kDoNotFlush };`.
See https://abseil.io/tips/94

Reply:
Sure, I like this!